### PR TITLE
Updates for API 2.09 including RSP1 and RSP2

### PR DESCRIPTION
**Pull request by SDRplay**

### CODE REVIEW

**Comment by cjcliffe** [comment]
@SDRplay seems to be working well on RSP1 here; but thought I'd check since there's no comment -- is it all ready to merge?

File: Settings.cpp [hunk 1]
```diff
@@ -277,248 +331,528 @@ std::vector<std::string> SoapySDRPlay::listFrequencies(const int direction, cons
...
+       setArgs.push_back(RfGainArg);
+    }
+    else
+    {
+       SoapySDR::ArgInfo RfGainArg;
+       RfGainArg.key = "rfgain_sel";
+       RfGainArg.value = "1";
+       RfGainArg.name = "RF Gain Select";
+       RfGainArg.description = "RF Gain Select";
+       RfGainArg.type = SoapySDR::ArgInfo::INT;
```
Comment (cjcliffe) [inline Settings.cpp:629]: Fixed this arg type in SDRPlay-master branch

`RfGainArg.type = SoapySDR::ArgInfo::STRING;`

Comment (SDRplay) [inline Settings.cpp]: Yes, all ready

**Comment by SDRplay** [comment]
Yes, all ready

**Comment by cjcliffe** [comment]
All merged now; thanks!

**Pull request merged**